rules/builds.smk

```diff
@@ -46,6 +46,85 @@ rule filter:
             --output {output.sequences} 2>&1 | tee {log}
         """
 
+rule excluded_sequences:
+    message:
+        """
+        Generating fasta file of exluded sequences
+        """
+    input:
+        sequences = rules.download.output.sequences,
+        metadata = rules.download.output.metadata,
+        include = config["files"]["exclude"]
+    output:
+        sequences = "results/excluded.fasta"
+    log:
+        "logs/excluded.txt"
+    conda: config["conda_environment"]
+    shell:
+        """
+        augur filter \
+            --sequences {input.sequences} \
+            --metadata {input.metadata} \
+            --include {input.include} \
+            --min-length 50000 \
```

Comment:
Can we delete this line? If the value is set so high as to disable the filter, we should be able to remove it.

Reply:
Sorry, I meant to highlight the line is `--min-length 50000 \`.

Reply:
yes, just did. needed to check first what are required arguments...

Reply:
this actually happens to be necessary. exclude everything and add the exclude file content.